### ISSUE TITLE
Add Alpine 3.8 images

### ISSUE THREAD

**Issue by MichaelSimons** [dotnet/dotnet-docker#747]
Add Alpine 3.8 images for 2.1 and newer.

This is currently blocked on https://github.com/dotnet/core-eng/issues/3874

**Comment by MichaelSimons** [comment]
This is being superseded by #921.  Also related is the [announcement ](https://github.com/dotnet/dotnet-docker/issues/933) on supporting new Alpine versions
